src/material-examples/material-module.ts

```diff
@@ -40,7 +40,8 @@ import {
     MdToolbarModule,
     MdTooltipModule,
     MdPaginatorModule,
-    MdSortModule
+    MdSortModule,
+    MdTableModule
```

Comment:
Looks like this adds `MdTableModule` for a second time (it's already defined on line 39).